modules/nf-core/sickle/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: "sickle"
+description: A windowed adaptive trimming tool for FASTQ files using quality
+keywords:
+  - fastq
+  - sliding window
+  - trimming
+tools:
+  - "sickle":
+      description: "a tool that determines clipping of reads on 3' end and 5'end based on base quality "
+      homepage: "https://github.com/najoshi/sickle"
+      documentation: "None"
+      tool_dev_url: "https://github.com/najoshi/sickle"
+      licence: "['MIT License']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false, qual:'sanger'  ]
+  - reads:
+      type: file
+      description: List of input FastQ files of size 1 and 2 for single-end and paired-end data, respectively.
```

Comment:
Can you also add here which file of the paired-end data has to be specified first?

Reply:
It is done in the updated PR.